client/src/App.tsx

```diff
@@ -51,6 +51,9 @@ const App = () => {
     // Events data for schedule
     const [events, setEvents] = useState<GCalEvent[] | null>(null);
 
+    // Reference to the favicon element
+    const [favicon, setFavicon] = useState<HTMLLinkElement | null>(null);
```

Comment:
Could use useRef here perhaps

Reply:
https://reactjs.org/docs/hooks-reference.html#useref

Reply:
Probably, I don't know react, but i found that the `<head>` element isn't a react node

Reply:
`head` doesn't have to be a React node. You can use `useRef` like

```ts
const favicon = useRef<HTMLLinkElement | null>(null);

useEffect(() => {
  if (!favicon.current) {
    favicon.current = document.createElement('link'); // etc
    document.head.append(favicon.current);
  }
  // use favicon.current
  return () => {
    if (favicon.current) {
      favicon.current.remove();
    }
  };
}, [])
```

A benefit of this is that if you use `useState`, if `favicon` changes, your `useEffect` would be dealing with the old favicon because React things